src/Symfony/Component/Cache/Adapter/FilesystemAdapter.php

```diff
@@ -126,7 +126,7 @@ protected function doDelete(array $ids)
     protected function doSave(array $values, $lifetime)
     {
         $ok = true;
-        $expiresAt = $lifetime ? time() + $lifetime : PHP_INT_MAX;
+        $expiresAt = time() + ($lifetime ?: 31557600); // 31557600s = 1 year
```

Comment:
I'd prefer to keep the number. To be completely self-explanatory, the constant should be named `self::ONE_YEAR_IN_SECONDS` or something like that.